### DISCUSSION TITLE
Third parties included in "netbeans12.6"

### DISCUSSION THREAD

**Discussion by 2020122300003133** [apache/netbeans#3577] [Q&A]
I would like to know which third parties are included in "netbeans12.6" and what their licenses are.

[https://netbeans.apache.org/wiki/DevFaqLic3rdPartyComponents.asciidoc](https://netbeans.apache.org/wiki/DevFaqLic3rdPartyComponents.asciidoc)
The linked pages in this page seem to have disappeared.

**Comment by vieiro** [comment]
D'oh. We now have bots demanding things in our discussion threads! 
Good to know these bots are not demanding pizzas, or beers, or money! :-D

**Comment by matthiasblaesing** [comment]
The binaries differ between versions, but each build archive contains a LICENSE file, which holds the licensing information for all contained files.

**Comment by vieiro** [comment]
And Apache NetBeans binary distributions include a NOTICE file too with all this information, as any other Apache top level project binary distribution does, and as mandated by the ASF Licensing howto https://infra.apache.org/licensing-howto.html

**Comment by 2020122300003133** [comment]
I'm sorry for the late reply.

I'm trying to find the folder where I downloaded the file, but I couldn't figure out exactly which one is the third party.
I will try to check more.

Thank you very much for your help.
